src/bindings/js/node/tests/unit/infer_request.test.js

```diff
@@ -300,3 +300,47 @@ describe('ov.InferRequest tests', () => {
     });
   });
 });
+
+describe('ov.InferRequest tests with missing outputs names', () => {
+  const modelV3Small = testModels.modelV3Small;
+  let compiledModel = null;
+  let tensorData = null;
+  let tensor = null;
+  let inferRequest = null;
+
+  before(async () => {
+    await isModelAvailable(modelV3Small);
+
+    const fs = require('fs');
```

Comment:
Please, move this require at the top of the module. Also, I suggest to use async version of the `readFile` method.